core/sql/pom.xml.apache

```diff
@@ -0,0 +1,131 @@
+<?xml version="1.0"?>
```

Comment:
Generally, profiles are used in Maven to build different versions, but profiles may not be sufficient for this case. See http://maven.apache.org/guides/introduction/introduction-to-profiles.html